### PR TITLE
Create das compliance report view

### PR DESCRIPTION
**Pull request by witpo**

### CODE REVIEW

File: src/SFA.DAS.DATA.MANAGEMENT/Views/Comp_DAS_Compliance_Report .sql [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+		,
+		CASE	
+			WHEN cp.CalendarMonth BETWEEN 1 AND 4 THEN cp.CalendarMonth + 8
+			ELSE cp.CalendarMonth - 4
+		END AS PayrollMonth	
+		,
+		CASE	
+			WHEN cp.CalendarMonth BETWEEN 1 AND 4 THEN CONCAT(CAST(RIGHT(cp.CalendarYear, 2) -1 AS VARCHAR), '-', CAST(RIGHT(cp.CalendarYear, 2) AS VARCHAR))
+			ELSE CONCAT(CAST(RIGHT(cp.CalendarYear, 2) AS VARCHAR),'-', CAST(RIGHT(cp.CalendarYear, 2) + 1 AS VARCHAR))
+		END AS PayrollYear
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/Views/Comp_DAS_Compliance_Report .sql:46]: Same Here, The Calculation seems to be deriving Academic Year/Month and not Payroll Year/Month. Is this Intended?
Alternatively, DASCalendarMonth table can be used to derive Payroll(Tax) Year/Month by joining on Calendar Year & Calendar Month Number.

File: src/SFA.DAS.DATA.MANAGEMENT/Views/Comp_DAS_Compliance_Report .sql [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+(
+	SELECT DISTINCT		
+		 CASE	
+		 	WHEN LD.PayrollMonth BETWEEN 1 AND 8 THEN LD.PayrollMonth + 4
+		 	ELSE LD.PayrollMonth - 8
+		 END AS CalendarMonth -- align	
+		,CASE	
+			WHEN LD.PayrollMonth BETWEEN 1 AND 8 THEN CONCAT('20', LEFT(LD.PayrollYear, 2))
+			ELSE CONCAT('20', FORMAT(CONVERT(INT, SUBSTRING(LD.PayrollYear, 4, 2)), '00'))
+		END AS CalendarYear
```
[the remark this answers is not in the document]
Comment (uhimabindu) [reply src/SFA.DAS.DATA.MANAGEMENT/Views/Comp_DAS_Compliance_Report .sql]: The New changes looks good to me now.

File: src/SFA.DAS.DATA.MANAGEMENT/Views/Comp_DAS_Compliance_Report .sql (hunk 1 above)
Comment (uhimabindu) [reply src/SFA.DAS.DATA.MANAGEMENT/Views/Comp_DAS_Compliance_Report .sql]: The New Changes looks good to me now.

**Review by uhimabindu — APPROVED** [review]
Approved.

**Review by sarmaevanidfe — APPROVED** [review]
Looks fine to me

**Review by arvinthseran — APPROVED** [review]

**Pull request merged**